### ISSUE TITLE
[WCAG Audit] [Unique Scenario ID] [Category - Product Name - Scenario Name] Title of the Bug

### ISSUE THREAD

**Issue by vaibhavsharma2611** [vaibhavsharma2611/vaibhavsharma2611#9]
**Important:** **Do not transfer** this issue into another repository. Instead, you can link to this issue from a project board.

### [MAS] (Microsoft Accessibility Standards) failure number and link to MAS guidance for remediation
()

### Helpful information for remediation
Link [Deque University article]() to help with remediation.

## Test Environment
- **OS:** iOS (16.6)
- **Device:** iPhone 13
- **Application:** GitHub Mobile (**Version:** 1.123.0)
- **Screen Reader:** VoiceOver
- **Luminance Checker Tool:** CCA - Colour Contrast Analyser (Version 3.3.0) (https://www.tpgi.com/color-contrast-checker/)
- **User Scenario Unique ID:** 
Link to the [User Scenarios Spreadsheet]()
